.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,53 @@
+name: test
+
+on:
+  schedule:
```

Comment:
Do we really want to run tests on a schedule when the code doesn't change? That does not seem sensible, given we do not interact with outside services.

Reply:
Because external dependences may change. So we see this if schedule tests fails some day.